include/ncengine/ui/ImGuiUtility.h

```diff
@@ -223,15 +229,27 @@ inline auto InputInt(int& value, const char* label, int step) -> bool
     return ImGui::InputInt(label, &value, step);
 }
 
+inline auto InputU8(uint8_t& value, const char* label) -> bool
+{
+    constexpr uint8_t step = 1;
+    return ImGui::InputScalar(label, ImGuiDataType_U8, &value, &step);
+}
+
+inline auto InputU16(uint16_t& value, const char* label) -> bool
+{
+    constexpr uint16_t step = 1;
+    return ImGui::InputScalar(label, ImGuiDataType_U16, &value, &step);
+}
+
 inline auto InputU32(unsigned& value, const char* label) -> bool
 {
-    constexpr auto step = 1u;
+    constexpr uint32_t step = 1;
     return ImGui::InputScalar(label, ImGuiDataType_U32, &value, &step);
 }
 
 inline auto InputU62(unsigned& value, const char* label) -> bool
```

Comment:
Missed before, whenever this was originally made - but the method name should be InputU64

Reply:
Oh boy, it also takes unsigned instead of uint64_t.